### PR TITLE
Create GET comments functionality and tests, amend GET articles functionality to include articles without comments

### PR DESCRIPTION
**Pull request by hsnnkb-dev**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -108,4 +107,57 @@ describe('api', () => {
...
+              .get('/api/articles/1/comments')
+              .expect(200)
+              .expect('Content-Type', 'application/json; charset=utf-8')
+              .then(({ body }) => {
+                const comments = body.comments;
+                expect(comments.length).toBe(11);
+                expect(comments).toBeSortedBy('created_at', { descending : true })
+                })
+    });
+
```
Comment (B7Bprog) [inline __tests__/app.test.js:143]: Please add a test for no comments for valid article ID, expecting 200 and an empty array.

**Review by B7Bprog — CHANGES_REQUESTED** [review]
Hey Hasan, please see my comment and add that extra test, also please remove all '.only' in your tests.
